includes/widgets/tabs.php

```diff
@@ -74,6 +74,10 @@ public function get_keywords() {
 		return [ 'tabs', 'accordion', 'toggle' ];
 	}
 
+	public function show_in_panel(): bool {
```

Comment:
If the experiment is off, shouldn't it not exist at all?

Reply:
we cannot unregister the module, doing so will not render the old tabs